x-pack/plugins/stack_connectors/common/gen_ai/constants.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const GEN_AI_TITLE = i18n.translate(
+  'xpack.stackConnectors.components.genAi.connectorTypeTitle',
+  {
+    defaultMessage: 'Generative AI',
+  }
+);
+export const GEN_AI_CONNECTOR_ID = '.gen-ai';
```

Comment:
When importing ala:
```import { GEN_AI_CONNECTOR_ID } from '@kbn/stack-connectors-plugin/common/gen_ai/constants';```

I'm seeing this error:


<img width="1733" alt="image" src="https://github.com/elastic/kibana/assets/2946766/496a261e-a041-4f3d-935a-0fa3ca9e018b">

I had this same issue when attempting to import some of the `trigger_actions_ui` exports via module import. For these constants be consumed by other packages, I think we'll need to either move this to a `public/common` export, or to add an `extraPublicDirs`'s entry to the `stack_connectors/kibana.jsonc`  configuration, ala: 
```
    "extraPublicDirs": [
      "public/common",
      "public/common/constants"
    ]
```
---

I'm not sure of the preferred approach here, so may want to ask the ops folks, but I imagine exporting via `public/common` should be 👍.